### PR TITLE
two test cases with multflt keyword

### PR DESCRIPTION
**Pull request by tskille**
Two additinal test model using MULTFLT keyword with faults referenced with asterisk (*).  Example 
```
MULTFLT
 'FLT1*' 0.001 /
 'FLT2*' 0.01 /
 'FLT3*' 0.05 /
 'FLT4*' 0.05 /
/
```

### CODE REVIEW

File: mult/MULTFLT-03.DATA [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+--  MULTFLT
+--   'FLT1A' 0.033 /
+--   'FLT*' 0.001 /
+--   'FLT*' 0.01 /
+--  /
+
+-- If a fault is referenced in more that one record in the MULTFLT keyword, 
+-- the last entry should be selected. Hence the MULTFLT keyword can be 
+-- spesified like this.
```
Comment (akva2) [inline mult/MULTFLT-03.DATA:37]: specified

File: mult/MULTFLT-03.DATA [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+-- an asterisk (*). In this way more that one fault multiplier can be set
+-- with one record.
+
+-- The model includes a total of 6 faults FLT1A, FLT1B, FLT2A, FLT2B, FLT3A
+-- and FLT4A.
+
+-- According to documentation, a fault name ending with asterisk (*) can
+-- select multiple faults. 
+
+-- In this deck the fallowing MULTFLT keyword is used.
```
Comment (blattms) [inline mult/MULTFLT-03.DATA:18]: There is a small typo. Should be "following".

**Comment by tskille** [comment]
both typos are fixed, thanks.

**Comment by blattms** [comment]
Perfect. Thanks a lot. I am merging this.

**Pull request merged**